generators/client/templates/angular/tsconfig.spec.json.ejs

```diff
@@ -16,8 +16,18 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-module.exports = {
-    plugins: [
-        require('autoprefixer')
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "outDir": "./out-tsc/spec",
```

Comment:
This folder should be added to gitignore/eslintignore/prettierignore?

Reply:
It's quite strange as I don't see any files written on the disk when I run unit tests...
so I don't know if we need to do this

Reply:
Maybe change to
```suggestion
        "outDir": "<%= BUILD_DIR %>out-tsc/spec",
```